test/test_morley.py

```diff
@@ -0,0 +1,25 @@
+import FIAT
+import finat
+import numpy as np
+from gem.interpreter import evaluate
+from fiat_mapping import MyMapping
+
+
+def test_morley():
```

Comment:
Morley is unrelated to this pull request, nevertheless, better test coverage is welcome.

Reply:
Yes, I was doing it as a sanity check for using the gem interpreter in tests.  Unrelated, but let's keep it in.